src/lib/components/SelectField/README.mdx

```diff
@@ -240,6 +237,70 @@ Full-width fields span the full width of a parent:
   }}
 </Playground>
 
+## Grouping Related Options
+
+In case of a large amount of options, consider **grouping related items
+together** by nesting them (implements the `optgroup` HTML element).
```

Comment:
Now when I see it live and in context, I'd change the texts after all. I believe this will work better:

1. Please keep the original hint in the _General Guidelines_ section, just shorter. The advice still has its place there.

   ```
   In case of a large amount of options, consider **grouping related items
   together** by nesting them.
   ```

2. Please change the text here so it works more like API description:

   ```suggestion
   For a large amount of options you can group related items together
   by nesting them (implements the `optgroup` HTML element).
   ```